src/util/Constants.js

```diff
@@ -521,14 +521,15 @@ exports.Colors = {
   DEFAULT: 0x000000,
   WHITE: 0xffffff,
```

Comment:
Add Discord's black?
```suggestion
  WHITE: 0xffffff,
  BLACK: 0x23272a,
```